controllers/keda/util/util.go

```diff
@@ -70,11 +70,22 @@ func UpdateKedaControllerStatus(ctx context.Context, cl client.Client, kedaContr
 
 func RunningOnOpenshift(ctx context.Context, logger logr.Logger, cl client.Client) bool {
 	gvk := schema.GroupVersionKind{Group: "route.openshift.io", Version: "v1", Kind: "route"}
+	return isGvkPresent(ctx, logger, cl, gvk)
+}
+
+// RunningWithMonitoringStack returns true if the monitoring stack (i.e. Prometheus) is present, false otherwise
```

Comment:
I think this should be `MonitoringCRDsPresent`. If the podmonitor and servicemonitor CRDs are present, we should create the objects, and if not, we can skip it. Whether prometheus is installed doesn't really matter to us. If it is, then great, or if something else is on the cluster which makes use of podmonitors / servicemonitors, then they can consume our objects. And if the CRDs exist, but nothing consumes the resources, that's fine too... it won't cause any problems.

Reply:
Makes sense. Addressed in https://github.com/kedacore/keda-olm-operator/pull/180/commits/5694b4d0633886505bcfcc8a03fb75bb38ac53b1

I felt `MonitoringCRDsPresent` is too generic and may imply other resources, so I proposed two specific checks for the CRDs in question.  @joelsmith  PTAL.